baja/src/baja.rs

```diff
@@ -283,80 +345,106 @@ impl Baja {
             )
         };
 
-        let common_name = {
-            // This should be randomly generated as below. But this is causing
-            // temporary problems on Trustzone, so instead of randomly generating
-            // it, we're using a static value for now.
-            // This does not compromise the security of the system
-            //let mut temp = vec![0; 3];
-
-            //let rng = ring::rand::SystemRandom::new();
-            //rng.fill(&mut temp)
-            //.map_err(|_| "Error generating random bytes")?;
+        // NOTE: this should be randomly generated as below. But this is causing
+        // temporary problems on TrustZone, so instead of randomly generating
+        // it, we're using a fixed value for now.  This **does not** compromise
+        // the security of the system in any way.
+        let name = {
+            // let mut temp = vec![0; 3];
+            // let rng = SystemRandom::new();
+            // rng.fill(&mut temp).map_err(|_| "Error generating random bytes")?;
             // It must be a valid DNS name, which must not be all numeric
             // so we add an a at the beginning to be sure
-            //let full_string = format!("a{:02x}{:02x}{:02x}", temp[0], temp[1], temp[2]);
-            //full_string[..7].to_string()
-            "ac40a0c".to_string()
+            // let full_string =
+            //   format!("a{:02x}{:02x}{:02x}", temp[0], temp[1], temp[2]);
+            // full_string[..7].to_string()
```

Comment:
I think the comment here should be in one continuous section as a suggested implementation strategy. Maybe just move all out of `{...}`and change the code to `let name = "ac40a0c".to_string();`.